manager/integration/tests/test_uninstall.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import os
+import time
+
+from common import SETTING_DELETING_CONFIRMATION_FLAG
+from common import batch_v1_api, core_api, get_longhorn_api_client # NOQA
+from common import RETRY_COUNTS, RETRY_INTERVAL_LONG
+
+@pytest.mark.uninstall   # NOQA
+def test_uninstall(batch_v1_api, core_api): # NOQA
+
+    client = get_longhorn_api_client()
+    lh_raw = "https://raw.githubusercontent.com/longhorn/longhorn"
+
+    longhorn_version = os.getenv("LONGHORN_REPO_BRANCH")
+
+    if longhorn_version == "master" or longhorn_version == "v1.4.x":
+        delete_confirmtion_flag = \
+            client.by_id_setting(SETTING_DELETING_CONFIRMATION_FLAG)
+        setting = client.update(delete_confirmtion_flag, value="true")
+        assert setting.name == SETTING_DELETING_CONFIRMATION_FLAG
+        assert setting.value == "true"
+
+    command = "kubectl create -f {}/{}/uninstall/uninstall.yaml".\
+              format(lh_raw, longhorn_version)
+
+    os.system(command)
+
+    uninstall_job_ready = False
+    for i in range(RETRY_COUNTS):
+        job = batch_v1_api.list_namespaced_job("longhorn-system",
+                                               label_selector="\
+                                               job-name=longhorn-uninstall")
+        if job.items[0].status.succeeded == 1:
+            uninstall_job_ready = True
+            break
+        time.sleep(RETRY_INTERVAL_LONG)
+    assert uninstall_job_ready is True
+
+    command = "kubectl delete -f {}/{}/deploy/longhorn.yaml"\
+              .format(lh_raw, longhorn_version)
+    os.system(command)
+
+    command = "kubectl delete -f {}/{}/uninstall/uninstall.yaml"\
+              .format(lh_raw, longhorn_version)
+    os.system(command)
+
+    longhorn_ns_deleted = False
+    for i in range(RETRY_COUNTS):
+        namespaces = core_api.list_namespace()
+        find = False
```

Comment:
`find` is too general. Having a good habit to make the variable name meaningful. Also, why not just using one variable instead of two in this case?

```python
longhorn_deleted = False

for i in range(RETRY_COUNTS):
    namespaces = core_api.list_namespace()

    for namespace in namespaces.items:
        if namespace.metadata.name == "longhorn-system":
            longhorn_deleted = False
            break

    if not longhorn_deleted:
        time.sleep(RETRY_INTERVAL_LONG)

assert longhorn_deleted
```